gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/JobCatalog.java

```diff
@@ -63,11 +65,16 @@ default StandardMetrics getStandardMetrics() {
    **/
   JobSpec getJobSpec(URI uri) throws JobSpecNotFoundException;
 
-  public static class StandardMetrics implements JobCatalogListener, StandardMetricsBridge.StandardMetrics {
+  @Slf4j
+  public static class StandardMetrics extends DefaultStandardMetrics implements JobCatalogListener {
```

Comment:
In the other classes, name `InnerStandardMetrics` is used. For readability, could we make the name consistent?

Reply:
This is a public interface. It is used by other LinkedIn MP, so better not change here for compatibility purpose.